### DISCUSSION TITLE
6700xt unlocks new 3060 on lolminer

### DISCUSSION THREAD

**Discussion by Dimmestlight** [Lolliedieb/lolMiner-releases#687] [General]
I'm using the newest NVIDIA drivers on hiveos and the latest hiveos build with lolminer 1.31. I am getting 34mh/s on 3060s that cannot run the beta NVIDIA bios or any other hashrate unlocking bios. On any other miner I get around 22mh/s on these cards and without the 6700xt mining with them I get 18mh/s on these cards. I tried setting it up to have 3060s only mining with lolminer and the 6700xt mining with teamredminer and it also dropped hashrate. I hope people who have multiple cards see this so they can partially unlock the newer locked down 3060s.

**Comment by hopium1986** [comment]
I think, for LHR v2 at this moment optimal algo is autolykos or kawpow.
